### PR TITLE
Refactor signup form and add subscribed page

### PR DESCRIPTION
**Pull request by khalilcodes**
Tasks from #77 

### Summary

This PR refactors the signup form component and adds a custom 'thank you for subscribing' page on form submission.

### Changes

* Remove Brevo html from signup form component 
* Add subscribed page in `pages/thank-you-for-subscribing.jsx`
* Configure Brevo settings to redirect to subscribed page after submit.

### Preview

* Subscribed page - https://d9ee7b83.diginomics.pages.dev/thank-you-for-subscribing

localhost_3000_thank-you-for-subscribing

### CODE REVIEW

**Comment by khalilcodes** [comment]
@rufuspollock should we make the signup form component reusable if it's going to be used in more than one place eg. like `<SignupForm buttonText="Sign up" buttonColor="bg-yellow-500" className="flex-row" />` ?

**Comment by rufuspollock** [comment]
> @rufuspollock should we make the signup form component reusable if it's going to be used in more than one place eg. like `<SignupForm buttonText="Sign up" buttonColor="bg-yellow-500" className="flex-row" />` ?

Not for now - YAGNI. Let's see what happens and then refactor if we see reuse.

File: pages/thank-you-for-subscribing.jsx
Comment (rufuspollock) [inline pages/thank-you-for-subscribing.jsx:1]: Personally i think we could have this even simpler and have this as markdown but i'm not fussed for now.

File: components/SignupForm.jsx
Comment (rufuspollock) [inline components/SignupForm.jsx:1]: Is this even worth keeping as a component? Let's leave for now but for future maybe we just want docs on how to create these.

**Review by rufuspollock — APPROVED** [review]

**Pull request merged**